RealmSwift.podspec

```diff
@@ -0,0 +1,28 @@
+Pod::Spec.new do |s|
```

Comment:
Can you really not define two specs in the same file? We could avoid duplicating a lot of this content.